mne/channels/tests/test_montage.py

```diff
@@ -1333,17 +1333,25 @@ def test_get_montage():
     # the montage does not change
     assert_object_equal(montage.dig, test_montage.dig)
 
+    # the montage should fulfill a roundtrip with make_dig_montage
+    test2_montage = make_dig_montage(**montage.get_positions())
+    assert_object_equal(test2_montage.dig, test_montage.dig)
+
     # 2. now do a standard montage
     montage = make_standard_montage('mgh60')
     # set the montage; note renaming to make standard montage map
-    raw.set_montage(montage)
+    raw.set_montage(montage.copy())
```

Comment:
You should not need to `.copy`, if attributes are not properly copied when setting we have a bug somewhere. Is this leftover debugging cruft?